cli/admin/src/main/java/io/pravega/cli/admin/utils/CLIControllerConfig.java

```diff
@@ -24,11 +24,14 @@
         SEGMENTSTORE, ZOOKEEPER
     }
 
-    private static final Property<String> CONTROLLER_REST_URI = Property.named("controller.rest.uri", "http://localhost:9091");
-    private static final Property<String> CONTROLLER_GRPC_URI = Property.named("controller.grpc.uri", "tcp://localhost:9090");
+    private static final Property<String> CONTROLLER_REST_URI = Property.named("controller.rest.uri", "localhost:9091");
```

Comment:
Some of these config names are inconsistent with the scheme we use elsewhere in Pravega. See the following: 
* [Configuration Naming Guidelines](https://github.com/pravega/pravega/wiki/Configuration-Naming-Guidelines)
* [Existing examples](https://github.com/pravega/pravega/wiki/Configuration-Name-Changes) 

Here are the suggested changes: 

* `controller.rest.uri` -> `controller.connect.rest.uri` (because it is connecting to the Controller's REST URI)
* `controller.grpc.uri` -> `controller.connect.grpc.uri`
* `security.auth.credentials.username` -> `controller.connect.credentials.username`
* `security.auth.credentials.password` -> `controller.connect.credentials.pwd` (we avoid the term `password` so that scanners and other tools don't get confused)
* `security.auth.token.signingKey` -- ? (what is this for, here?)

Etc.

As for `security.auth.enable` (and similar items), it indicates you are enabling auth for CLI. If you are using it for specifying that auth be enabled for CLI to controller communication, the config name'd be different.

Reply:
I've used the guidelines.